services/efs-guardian/Dockerfile

```
@@ -0,0 +1,192 @@
+ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
+
+EXPOSE 8000
+EXPOSE 3000
+
+# -------------------------- Build stage -------------------
+# Installs build/package management tools and third party dependencies
+#
+# + /build             WORKDIR
+#
+FROM base as build
+
+ENV SC_BUILD_TARGET=build
+
+RUN --mount=type=cache,target=/var/cache/apt,mode=0755,sharing=private \
+  --mount=type=cache,target=/var/lib/apt,mode=0755,sharing=private \
+  set -eux \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends \
+  build-essential
+
+# NOTE: install https://github.com/astral-sh/uv ultra-fast rust-based pip replacement
+RUN --mount=type=cache,mode=0755,target=/root/.cache/pip \
+  pip install uv~=0.1
+
+# NOTE: python virtualenv is used here such that installed
+# packages may be moved to production image easily by copying the venv
+RUN uv venv "${VIRTUAL_ENV}"
+
+RUN --mount=type=cache,mode=0755,target=/root/.cache/uv \
+  uv pip install --upgrade  \
+  pip~=24.0  \
+  wheel \
+  setuptools
+
+WORKDIR /build
+
+# install base 3rd party dependencies
+# NOTE: copies to /build to avoid overwriting later which would invalidate this layer
+RUN \
+  --mount=type=bind,source=services/efs-guardian/requirements/_base.txt,target=_base.txt \
+  --mount=type=cache,mode=0755,target=/root/.cache/uv \
+  uv pip install \
+  --requirement _base.txt
+
+
+# --------------------------Prod-depends-only stage -------------------
+# This stage is for production only dependencies that get partially wiped out afterwards (final docker image concerns)
+#
+#  + /build
+#    + services/efs-guardian [scu:scu] WORKDIR
+#
+FROM build as prod-only-deps
+
+ENV SC_BUILD_TARGET prod-only-deps
+
+WORKDIR /build/services/efs-guardian
+
+RUN \
+  --mount=type=bind,source=packages,target=/build/packages,rw \
+  --mount=type=bind,source=services/efs-guardian,target=/build/services/efs-guardian,rw \
+  --mount=type=cache,mode=0755,target=/root/.cache/uv \
+  uv pip install \
+  --requirement requirements/prod.txt \
+  && uv pip list
+
+
+# --------------------------Production stage -------------------
+# Final cleanup up to reduce image size and startup setup
+# Runs as scu (non-root user)
+#
+#  + /home/scu     $HOME = WORKDIR
+#    + services/efs-guardian [scu:scu]
+#
+FROM base as production
+
+ENV SC_BUILD_TARGET=production \
+  SC_BOOT_MODE=production
+
+ENV PYTHONOPTIMIZE=TRUE
+
+WORKDIR /home/scu
+# ensure home folder is read/writable for user scu
+RUN chown -R scu /home/scu
+
+# Starting from clean base image, copies pre-installed virtualenv from prod-only-deps
+COPY --chown=scu:scu --from=prod-only-deps  ${VIRTUAL_ENV} ${VIRTUAL_ENV}
+
+# Copies booting scripts
+COPY --chown=scu:scu services/efs-guardian/docker services/efs-guardian/docker
+RUN chmod +x services/efs-guardian/docker/*.sh
+
+
+HEALTHCHECK --interval=10s \
+  --timeout=5s \
+  --start-period=5s \
+  --retries=5 \
+  CMD ["python3", "services/efs-guardian/docker/healthcheck.py", "http://localhost:8000/"]
```

Please use this one, so that the service starts faster
```suggestion
HEALTHCHECK \
  --interval=10s \
  --timeout=5s \
  --start-period=30s \
  --start-interval=1s \
  --retries=5 \
  CMD ["python3", "services/efs-guardian/docker/healthcheck.py", "http://localhost:8000/"]
```